src/FacebookEventForwarder.js

```diff
@@ -216,6 +219,24 @@ var name = 'Facebook',
                     }
                     else if (event.ProductAction.ProductActionType == mParticle.ProductActionType.AddToCart){
                         eventName = ADD_TO_CART_EVENT_NAME;
+
+                        // Set product names as content_name if enabled
+                        if (sendProductNamesAsContents) {
+                            productNames = buildProductNames(event.ProductAction.ProductList);
+                            if (productNames && productNames.length > 0) {
+                                params['content_name'] = productNames;
+                            }
+                        }
+
+                        if (event.ProductAction.TransactionId) {
+                            params['order_id'] = event.ProductAction.TransactionId;
+                        }
+                        
+                        // Build contents array for AddToCart events
+                        contents = buildProductContents(event.ProductAction.ProductList);
+                        if (contents && contents.length > 0) {
+                            params['contents'] = contents;
+                        }

```

Comment:
From a product (and code simplification/maintainability) perspective, I think that these new settings  should support all of the commerce events types.

The reason being - we have one customer who wants this to be in a few commerce types because those are the only ones they currently use.  But if another customer does use a separate commerce event type, they would expect the settings selected in the UI to apply to all commerce event types, not just a few.

So let's move logic related to `sendProductNamesAsContents` and `order_id` to above and outside of when we do specific commerce event typ elogic. (I'd say line 192.